package.json

```diff
@@ -61,6 +61,22 @@
   "dependencies": {
     "lodash": "^4.17.21"
   },
+  "peerDependencies": {
+    "swagger-ui-dist": "^5.10.5",
```

Comment:
The version constraint for swagger-ui-dist should be kept open for future major version upgrades.